From 452dc78678c9929f60f9b4e8162b44567ffabb7d Mon Sep 17 00:00:00 2001
From: Girish Sharma <girish.sharma@einfochips.com>
Date: Tue, 12 Apr 2016 18:53:06 +0530
Subject: [PATCH 3/3] Disable Signed check for boot image

This change will disable signed check functionality for boot.img

Change-Id: Iafbf2bcb1f06969627e68d28f9e630ff1a39b8da
Signed-off-by: Girish Sharma <girish.sharma@einfochips.com>
---
 BoardConfig.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index adf41c4..59f89dc 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -67,7 +67,7 @@ BOARD_TOMBSTONESIMAGE_PARTITION_SIZE := 268435456
 BOARD_FLASH_BLOCK_SIZE := 131072 # (BOARD_KERNEL_PAGESIZE * 64)
 
 # Use signed boot and recovery image
-TARGET_BOOTIMG_SIGNED := true
+TARGET_BOOTIMG_SIGNED := false
 
 TARGET_USE_KRAIT_PLD_SET := true
 TARGET_KRAIT_BIONIC_PLDOFFS := 10
-- 
1.9.1

